docs/user/getting-started.md

```diff
@@ -55,8 +55,8 @@ REM Get the Base64 encoded token
 kubectl get secret %ks% -o jsonpath={.data.token} > b64.txt
 
 REM Decode The Token
-DEL token.txt
```

Comment:
I don't have a windows box to test on (though I'm sure I could online somewhere, but), this line looks as though it was intended to delete a previously generated `token.txt`, whereas in your change it looks like you understood it to be deleting the intermediate `b64.txt` file (which may also be valid).

Does `certutil -decode` fail if the output file already exists? (ie. if you run this script a second time).

I assume from your change that the failure you were seeing is that it can't delete a file that doesn't exist? (the first time this is run).